crates/syntax/fuzz/Cargo.toml

```diff
@@ -1,21 +1,17 @@
-
 [package]
 name = "syntax-fuzz"
 version = "0.0.1"
 publish = false
-
-authors.workspace = true
-edition.workspace = true
-license.workspace = true
-rust-version.workspace = true
+authors = ["Automatically generated"]
```

Comment:
Let's drop this, it's optional now.

Reply:
And let's add `rust-version` too. It shouldn't matter since it's set on `syntax`, but won't hurt.

Reply:
> Let's drop this, it's optional now.

I assume you mean the `authors` field, not `edition`?

Reply:
Huh, weird, the diff view displays your comments for the *next* line, in the comments view they are on the `authors` line.

Reply:
Yeah, I meant the authors.